Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Build a notary-server container without the build environment
+
+FROM ubuntu:14.04
+
+RUN apt-get update && \
+    apt-get install -y ca-certificates && \
```

Comment:
if all you need is ca-certificates you can 
1. build from scrach and include them here but that takes maintenence
2. use something smaller like alpine